api/src/main/java/jakarta/data/repository/Pageable.java

```diff
@@ -177,7 +177,7 @@ static Pageable ofSize(int maxPageSize) {
      * @param pageNumber The page number
      * @return a new instance of <code>Pageable</code>. This method never returns <code>null</code>.
      */
-    Pageable page(long pageNumber);
+    Pageable newPage(long pageNumber);
```

Comment:
Another option that I thought of for the fluent-style configuration could be `.withSize(max)`, `.withPage(num)`, `.withSorts(sorts)` but `.newSize(max)`, `.newPage(num)`, ... that you came up with is shorter.

Reply:
It is a good point.
I don't want to spend a lot of time on it, but I like the `with` a prefix.